asn1/README.md

```diff
@@ -0,0 +1,11 @@
+### Asn1 fuzzer
+
+Fuzzing example with honggfuzz for the asn1 crate.
+
+From the [asn1](https://crates.io/crates/asn1) readme:
+> This is a Rust library for parsing and generating ASN.1 data (DER only).
+
+
+## How to run
+
+Simply run the command `cargo hfuzz run asn1`
```

Comment:
you can mention in this section of the readmes in which folder you have to be in to run the command, in this case the folder is **asn1** but for the ones made with cargofuzz its the fuzz folder within the fuzzer

Reply:
Resolved: https://github.com/lambdaclass/fuzzing_examples/pull/16/commits/30deb212e89da93e4580217e917f3704eac4a5c4